### DISCUSSION TITLE
Robot arm doing inverse kinematics on a PC client

### DISCUSSION THREAD

**Discussion by jaguilar** [pybricks/support#1231] [Show and tell]
https://photos.app.goo.gl/Ab5XpyetLmJZGETm9

Hey I built this with the helpful feedback of folks on this forum and over on the Eurobricks forums.

There are definitely better and more creative Lego robot arms out there, but I'm proud of this as a starter project. Thank you for taking the time to help me with my questions! Looking forward to going through the EV3 discovery book with pybricks next.

**Comment by jaguilar** [comment]
Source code is over here in case anyone sees anything useful in this: https://github.com/jaguilar/ev3test

Particularly, scipy makes IK trivial: https://github.com/jaguilar/ev3test/blob/main/kinematics.py

**Comment by dlech** [comment]
Thanks for sharing!
